gymroutine-mobile/ViewModels/ProfileViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ProfileViewModel.swift
+//  gymroutine-mobile
+//
+//  Created by 조성화 on 2025/01/02.
+//
+
+import Foundation
+
+@MainActor
+final class ProfileViewModel: ObservableObject {
+    @Published var user: User?
+    @Published var followersCount: Int = 0
+    @Published var followingCount: Int = 0
+
+    private let userManager: UserManager
+
+    init(userManager: UserManager) {
+        self.userManager = userManager
+        loadUserData()
+    }
```

Comment:
シングルトンの場合は、initでデータを渡さなくても、let宣言時に代入できるので短縮しちゃってください。

```suggestion
    private let userManager = UserManager.shared

    init() {
        loadUserData()
    }
```

Reply:
@yuki035 お願いします

Reply:
@hwacho9 
変更箇所を確認しました。
@sattsat-tdu のレビューに完全同意なので、修正しちゃってください

Reply:
@sattsat-tdu できそう？